### DISCUSSION TITLE
Add animate-shimmer class

### DISCUSSION THREAD

**Discussion by chirag04** [tailwindlabs/tailwindcss#3363] [Ideas]
Currently tailwind has animate-pulse which is great in itself but it can be bit overwhelming at time specially on grids/lists. Shimmer is a popular animation pattern for placeholders on a grid or list view. If tailwind can support that out-of-box that would be awesome!

**Comment by PeerRich** [comment]
this could be a great native class: https://twitter.com/delba_oliveira/status/1549426693357568001?s=12&t=Y7fP0FaYs5SfKQx2eEjhrg

**Comment by aryomuzakki** [comment]
this seems to be very old discussion, too bad no animate-shimmer added to v4

here is my version which make it reusable individual skeleton and adopting to v4, from the twitter post
https://play.tailwindcss.com/TvcbZjU6W5

maybe one day this will be updated to use pseudo element

**Reply by solodevx** [reply]
Tailwind v4 is really stressing me out especially with the new nextjs and turbo

**Comment by petekp** [comment]
check out https://www.assistant-ui.com/tw-shimmer !
